modules/rentbits/widgets/rb-widgets.js

```diff
@@ -0,0 +1,13 @@
+function makeRbFrame() {
+	rb_data.forEach(function(rb_widget, index) {
```

Comment:
__[jshint]__
* 'rb_data' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->